src/Symfony/Component/DependencyInjection/Tests/Fixtures/containers/container9.php

```diff
@@ -54,7 +55,7 @@
 ;
 $container->
     register('factory_service', 'Bar')->
-    setFactoryService('foo.baz')->
+    setFactoryService('%foo_baz_service%')->
```

Comment:
FWIW, I think it was a mistake to support this use case, but as it is, we should support it thoroughly.